SupersonicSound/LowLevel/Sound.cs

```diff
@@ -32,6 +33,24 @@ public static Sound FromFmod(FMOD.Sound sound)
             return new Sound(sound);
         }
 
+        public void Dispose()
```

Comment:
This can be simplified not to have the Dispose(true) and SuppressFinalize(this) boilerplate. The GC doesn't manage the lifetime of structs and so they will never be finalized!

It can literally just be:

    public void Dispose() { FmodSound.release(); }